### PR TITLE
bypass monkey patch on Rails 6.1 builds.

### PR DESCRIPTION
**Pull request by joelhawksley**
This change disables the monkey patch when the Rails version
is at least 6.1.

This change confirms the compatibility of the library with Rails 6.1.

### CODE REVIEW

**Comment by jonspalmer** [comment]
Do we have existing tests that exercise this?

**Comment by joelhawksley** [comment]
@jonspalmer we do! As we use matrix builds for CI, every test runs against `5.0.0`, `5.2.3`, `6.0.0`, and `master` 😄

File: lib/view_component/render_monkey_patch.rb [hunk 1]
```diff
@@ -3,7 +3,7 @@
 module ViewComponent
   module RenderMonkeyPatch # :nodoc:
     def render(options = {}, args = {}, &block)
-      if options.respond_to?(:render_in)
+      if options.respond_to?(:render_in) && Rails.version.to_f < 6.1
```
Comment (jonspalmer) [inline lib/view_component/render_monkey_patch.rb]: Should we do just a tiny bit of cleanup here? Perhaps a static method like `ViewComponent::apply_render_monkey_patch(options)` that we use is these two locations?

Comment (joelhawksley) [reply lib/view_component/render_monkey_patch.rb]: I agree that we should just wrap the monkey patch entirely once we ship `v2`. As I plan to do that in the next week or so, I'm not going to sweat this temporary duplication 😄

**Pull request merged**